examples/z_querier.py

```diff
@@ -0,0 +1,160 @@
+#
+# Copyright (c) 2024 ZettaScale Technology
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+# which is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+#
+# Contributors:
+#   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+#
+import itertools
+import time
+from typing import Optional, Tuple
+
+import zenoh
+
+
+def main(
+    conf: zenoh.Config,
+    selector: str,
+    target: zenoh.QueryTarget,
+    payload: str,
+    timeout: float,
+    iter: int,
```

Comment:
if `iter` is optional, it should be typed consequently. I know you pasted the code from `z_pub`, but you can fix `z_pub` at the same time.

Reply:
fixed